pkg/storage/unified/resource/table.go

```diff
@@ -547,3 +552,29 @@ func (x *resourceTableColumn) Decode(buff []byte) (any, error) {
 	}
 	return v, err
 }
+
+// AssertTableSnapshot will match a ResourceTable vs the saved value
+func AssertTableSnapshot(t *testing.T, path string, table *ResourceTable) {
```

Comment:
😢 I don't like that this moved out of the _test package so that i could use it in the other folder

I wonder if there is a way to have it work for tests outside the same package